packages/sdk/README.md

```diff
@@ -156,6 +156,187 @@ channel.banUser(message.sender);
+```
+
+To allow your users to block another users inside a private channel context use 
+
+```typescript
+const userBlocked: Boolean = await PrivateChannel.blockPrivateUser(currentUser, currentSite, targetUserId)
+```
+
+The same is valid to unblock another blocked user
+
+```typescript
+const userUnblocked: Boolean = await PrivateChannel.unblockPrivateUser(currentUser, currentSite, targetUserId)
+```
+
+To send a private message on a specific channel you can call the following method
+
+```typescript
+await peterChannel.sendMessage(
+  new ChatMessageContent(messageOptions)
+  replyMessageId, // optional
+  "temporaryId" 
+)
+```
+
+It's also possible to obtain an already created private channel directly throught the sdk by passing the `channelId`
+
+```typescript
+const aliceChannel: BasePrivateChannel = await arenaChat.getPrivateChannel("alice-channel")
+```
+
+In the same way we can list only the private channels that are not group channels for the current user
+
+```typescript
+const userPrivateChannels: BasePrivateChannel[] = await arenaChat.getUserPrivateChannel()
+```
+
+Also, it's possible to create a channel via `ArenaChat` instance or return an existing one for a userId
+
+```typescript
+const bobChannel: BasePrivateChannel = await arenaChat.createUserPrivateChannel(
+  "bobUserId",
+  new ChatMessageContent(messageParams) // optional
+)
+```
+
+About the event listeners this one deserves attention, types are defined here [types]('https://github.com/stationfy/arena-chat-sdk/tree/feature/sprint95-update-chat-sdk-doc/packages/types')
+
+- onUnreadMessagesCountChanged : Watchs for unread messages within a private channel so you can display it for the user in real time, callback will hold the unread count
+
+```typescript
+onUnreadMessagesCountChanged(user: ExternalUser, site: Site, callback: (total: number) => void): () => void 
+```
+
+- offUnreadMessagesCountChanged : From the sdk you can unsubscribe the a previous setted listener for `onUnreadMessagesCountChanged`
+
+```typescript
+arenaChat.offUnreadMessagesCountChanged()
+```
+
+### Polls
+
+Adding polls to engage the user experience is quite simple since it's enabled by your chat
+
+To configure a polls manager you need your previously configured `chatRoom` and a `arenaChat`, refers to [types](https://github.com/stationfy/arena-chat-sdk/tree/feature/sprint95-update-chat-sdk-doc/packages/types)
+
+```typescript
+import Polls from '@/polls/polls.ts'
+
+const polls: BasePolls = new Polls(chatRoom, arenaChat)
+```
+
+Once you get a polls instance it's possible to start loading the existing polls
+
+```typescript
+const pollsList: [Poll] = await polls.loadPolls(
+  PollFilter.RECENT, // optional
+  50 // optional
+)
+```
+
+To register a vote in a option for a poll you need to inform the `pollId` the `optionId` that is a number starting in 0 and optionaly an `anonymousId`
+
+```typescript
+await polls.pollVote(
```

Comment:
@cesarwbr i think passing the anonymousId here can be confusing for the sdk users. What happens if the user is already set?

Reply:
task created: https://app.clickup.com/t/fuuzhf